robosuite/environments/base.py

```diff
@@ -104,6 +106,7 @@ def __init__(
         render_camera="frontview",
         render_collision_mesh=False,
         render_visual_mesh=True,
+        render_gpu_device_id=-1,
```

Comment:
I assume -1 corresponds to no GPU (or maybe default?) -- can we make this explicit in the docstring?

Reply:
yeah it's inferred according to this [here](https://github.com/openai/mujoco-py/blob/master/mujoco_py/mjrendercontext.pyx#L104) - we can update the docstring